Includes/subsystems.cpp

```diff
@@ -1,16 +1,47 @@
 #include "subsystems.h"
 
 #ifdef NXDK
+#include <nxdk/mount.h>
+#include <windows.h>
+#include <pbkit/pbkit.h>
+#include <hal/input.h>
 #include <hal/video.h>
-#include <winapi/synchapi.h>
+#include <hal/xbox.h>
 #endif
 
+#include <SDL.h>
+#include <SDL_ttf.h>
+
+#include "outputLine.h"
+
 int init_systems() {
 #ifdef NXDK
   VIDEO_MODE xmode;
   void *p = NULL;
   while (XVideoListModes(&xmode, 0, 0, &p)) {}
   XVideoSetMode(xmode.width, xmode.height, xmode.bpp, xmode.refresh);
+
+  if (!nxMountDrive('C', "\\Device\\Harddisk0\\Partition2")) {
+    outputLine("Mounting error: Could not mount drive C\n");
+  }
+  if (!nxMountDrive('E', "\\Device\\Harddisk0\\Partition1")) {
+    outputLine("Mounting error: Could not mount drive E\n");
+  }
+  if (!nxMountDrive('F', "\\Device\\Harddisk0\\Partition6")) {
+    outputLine("Mounting error: Could not mount drive F\n");
+  }
+  if (!nxMountDrive('G', "\\Device\\Harddisk0\\Partition7")) {
+    outputLine("Mounting error: Could not mount drive G\n");
```

Comment:
Should this really be reported as an error, G is highly optional IIRC?

Reply:
I think it's fine for now, at least for logging purposes. It wouldn't present it to the user as an error, though.